css/pictures.css

```diff
@@ -19,11 +19,36 @@
 }
 
 .unsorted, .pic-box {
-  height: 190px;
   overflow-y: auto;
   overflow-x: hidden;
 }
 
+.pic-box {
+  height:249px;
```

Comment:
![screen shot 2015-04-09 at 10 17 26 pm](https://cloud.githubusercontent.com/assets/2756459/7080827/510419fc-df06-11e4-900d-73b48dd34c03.png)

250px


![screen shot 2015-04-09 at 10 17 53 pm](https://cloud.githubusercontent.com/assets/2756459/7080830/59ab7c76-df06-11e4-9ecb-4b1d68c8af61.png)

249 px


you can see the bottom border gets kinda wonky but i'll look into it more